Sources/UI/Chat/Cells/MessageTableViewCell+Updates.swift

```diff
@@ -81,14 +81,39 @@ extension MessageTableViewCell {
         messageLabel.text = text
     }
     
-    func enrichText(with message: Message, enrichURLs: Bool) {
+    func cachedEnrichText(with message: Message,
+                          enrichURLs: Bool) {
+        messageTextEnrichment = MessageTextEnrichment(message, style: style, enrichURLs: enrichURLs)
+
+        if let cached = messageTextEnrichment?.cachedEnrichment() {
+            self.messageLabel.attributedText = cached
+        }
```

Comment:
1. I don't this we should create `MessageTextEnrichment` to check the cache.
2. We can make cache as a static functions because it only depends on `cacheKey`
3. `cacheKey` now depends only on `text`, but additionally should be by `enrichURLs`
3.1. I don't think we should keep this attribute `enrichURLs`. It supposes to be useful with some styles, but it's internally always `true` and I think it was done prematurely.  I suggest to remove it at all.

So, below in the code I suggest do this:
```swift 
if let cachedEnrichedText = MessageTextEnrichment.cachedEnrichedText(for: message) {
    cell.messageLabel.attributedText = cachedEnrichedText
}
```